src/murmur/About.h

```diff
@@ -14,12 +14,21 @@
 # include <QtGui/QDialog>
 #endif
 
+enum AboutDialogOptions {
```

Comment:
Rather then iterating, why did you bit-flag them? For potential sub-tabbing and specifying two at the same time?